exec/jdbc-all/pom.xml

```diff
@@ -102,6 +102,18 @@
           <groupId>commons-codec</groupId>
           <artifactId>commons-codec</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>com.sun.jersey</groupId>
+          <artifactId>jersey-core</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.sun.jersey</groupId>
+          <artifactId>jersey-server</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.sun.jersey</groupId>
```

Comment:
hadoop 3.3.6 uses my fork of jersey-json -- see https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-common/3.3.6

Reply:
Ah! Thanks.